src/main/configurations/xxllnc/Configuration_Handler_AddDocumentToCaseCommand.xml

```diff
@@ -105,27 +105,66 @@
                 </IbisLocalSender>
                 <Param name="Alpha3" sessionKey="ZgwEnkelvoudigInformatieObject" xpathExpression="//taal" />
                 <Forward name="success" path="EnrichZgwEnkelvoudigInformatieObject" />
-            </SenderPipe>
+            </SenderPipe> -->
 
             <XsltPipe
                 name="EnrichZgwEnkelvoudigInformatieObject"
                 getInputFromSessionKey="ZgwEnkelvoudigInformatieObject"
                 styleSheetName="Zgw/Documenten/Model/ZgwEnkelvoudigInformatieObject.xslt"
                 storeResultInSessionKey="ZgwEnkelvoudigInformatieObject"
                 >
-                <Param name="Taal" xpathExpression="concat(concat($Alpha2, '_'), upper-case($Alpha2))">
+                <!-- <Param name="Taal" xpathExpression="concat(concat($Alpha2, '_'), upper-case($Alpha2))">
                     <Param name="Alpha2" sessionKey="Alpha2" />
-                </Param>
-                <Forward name="success" path="EXIT"/>
+                </Param> -->
+                <Forward name="success" path="CallWrapEdcLk01Request"/>
             </XsltPipe>
 
-            <!-- ZgwEnkelvoudigInformatieObject + ZgwZaak to object entiteitType="EDC" Translate(from ZaakBrug) -->
+            <!-- Create VoegZaakdocumentToe message -->
+            <SenderPipe 
+                name="CallWrapEdcLk01Request"
+                storeResultInSessionKey="EdcLk01RequestMessage">
+                <IbisLocalSender
+                    name="CallWrapEdcLk01RequestSender"
+                    javaListener="WrapEdcLk01Request"
+                    returnedSessionKeys="Error">
+                    <Param name="CorrelationId" sessionKey="CorrelationId"/>
+                    <Param name="ZgwEnkelvoudigInformatieObject" sessionKey="ZgwEnkelvoudigInformatieObject"/>
+                    <Param name="Base64Inhoud" sessionKey="ZgwEnkelvoudigInformatieObjectInhoud"/>
+                    <Param name="CaseReferenceNumber" sessionKey="CaseReferenceNumber" />
+                    <Param name="CaseDescription" sessionKey="CaseDescription" />
+                    <Param name="CaseTypeDescription" sessionKey="CaseTypeDescription" />
+                    <Param name="CaseTypeCode" sessionKey="CaseTypeCode" />
+                </IbisLocalSender>
+                <Forward name="success" path="VoegZaakdocumentToeSender"/>
+                <Forward name="exception" path="EXCEPTION"/>
+            </SenderPipe>
 
-            <!-- Wrap above in VoegZaakdocumentToe message with soapWrapperPipe -->
+            <SenderPipe
+                name="VoegZaakdocumentToeSender"
```

Comment:
Can't use EXCEPTION exit, as it will be considered a caught exception by ff and will trigger the errorstore.